modules/tinymce/src/themes/silver/test/ts/browser/editor/color/TextCustomColorMapBackgroundColorSanityTest.ts

```diff
@@ -0,0 +1,46 @@
+import { UiFinder } from '@ephox/agar';
+import { Assert, describe, it } from '@ephox/bedrock-client';
+import { SugarShadowDom } from '@ephox/sugar';
+import { TinyDom, TinyHooks, TinySelections, TinyUiActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import { getColorCols } from 'tinymce/themes/silver/ui/core/color/Options';
+
+describe('browser.tinymce.themes.silver.editor.color.TextCustomColorMapBackgroundColorSanityTest', () => {
```

Comment:
These three test are so similar I think they should be combined into one test file. You can just have three contexts with the hook in them so that you can share the `setupContent` and `assertUiElementDoesNotExist` between them. So stick them all into `TextCustomColorMapSanityTest.ts`?

Reply:
Can't, they use different color setups, which includes the col-calculations, which seems to be have been done ahead of time. Unless I have completely misunderstood something, making these tests requires them to be separate.

Reply:
Seems to work fine to combine them. The cols are calculated on registry of the option that is done on setup of a editor instance so as long as the configs and editor setup are within it's own context in the test they should be treated as they where separate files. The file separation is irrelevant even the test framework doesn't even know that there are multiple files all it sees is a long list of describes, context and it calls in a `scratch/compiled/tests-imports.ts` file bundle.

Reply:
- [ ] Combine the tests using `context` to remove duplication

Reply:
Putting them physically into the same file would be possible, but we have historically frowned upon putting what is, effectively, different tests in the same file. But very well, I'll do it since it was requested.

Reply:
I'd disagree with your statement there... I can't speak for Spocke, but from what he's said and what I've read this appears to be the same tests just with different options. In that case we do try and keep it together as it's more maintainable since it's all running via the same code.